<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAnimator</class>
 <widget class="QDialog" name="DialogAnimator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QSlider" name="verticalSlider_3">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="verticalSlider_2">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="13">
     <widget class="QSlider" name="verticalSlider_14">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="14">
     <widget class="QSlider" name="verticalSlider_15">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="verticalSlider_1">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QSlider" name="verticalSlider_5">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="15">
     <widget class="QSlider" name="verticalSlider_16">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QSlider" name="verticalSlider_9">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSlider" name="verticalSlider_4">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QSlider" name="verticalSlider_10">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QSlider" name="verticalSlider_7">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QSlider" name="verticalSlider_6">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QSlider" name="verticalSlider_11">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QSlider" name="verticalSlider_8">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="12">
     <widget class="QSlider" name="verticalSlider_13">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="11">
     <widget class="QSlider" name="verticalSlider_12">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="11">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="13">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="14">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="15">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="previousButton">
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>290</y>
     <width>109</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>sliderChanged()</slot>
 </slots>
</ui>
